ROLES.md

```diff
@@ -186,34 +152,37 @@ status is scoped to a part of the codebase.
 The following apply to the part of the codebase for which one would be an
 approver in an OWNERS file:
 
-- Reviewer of the codebase for at least 3 months.
+- Add yourself as a `writer` for the repositories you are responsible for
+  in [knative.yaml](../main/peribolos/knative.yaml) and/or
+  [knative-sandbox.yaml](../main/peribolos/knative-sandbox.yaml). After
+  this you should be an `approver` in the OWNERS file.
 
-- Should have reviewed or contributed to substantial PRs to the codebase.
+- Reviewer of the codebase for at least 3 months
 
-- Should be a `reviewer` in the OWNERS file.
+- Should have reviewed or contributed to substantial PRs to the codebase
 
-- Nominated by an a WG lead (with no objections from other leads).
+- Nominated by a Working Group Lead with no objections from other leads
 
 ### Responsibilities and privileges
 
 The following apply to the part of the codebase for which one would be an
 approver in an OWNERS file:
 
-- Approver status can be a precondition to accepting large code contributions.
+- Approver status may be a precondition to accepting large code contributions
```

Comment:
Can we strike this?  It's a "may", so there's no guarantee of enforcement, and I'd rather have that discussion with a particular code contribution in hand to consider.